src/ssort/_parsing.py

```diff
@@ -128,6 +128,16 @@ def split_class(statement):
     assert token.type == NAME
 
     token = next(tokens)
+    if token.string == "[":
```

Comment:
I've added this to parse the type parameter in class definitions. AFAIK the brackets cannot be nested, but I've copied the same structure used for parsing the bases between parenthesis.